tests/densify/test_interface_patchmatchnet.py

```diff
@@ -0,0 +1,108 @@
+"""Unit tests for the interface for PatchmatchNet
+
+Authors: Ren Liu
+"""
+import unittest
+from pathlib import Path
+
+import numpy as np
+from gtsam import PinholeCameraCal3Bundler
+
+from gtsfm.common.gtsfm_data import GtsfmData, SfmTrack
+from gtsfm.loader.olsson_loader import OlssonLoader
+from gtsfm.densify.interface_patchmatchnet import PatchmatchNetData
+
+DATA_ROOT_PATH = Path(__file__).resolve().parent.parent / "data"
+
+DEFAULT_FOLDER = DATA_ROOT_PATH / "set1_lund_door"
+
+NUM_VIEWS = 5
+
+EXAMPLE_CAMERA_ID = 1
+
+
+class TestInterfacePatchmatchNet(unittest.TestCase):
```

Comment:
the name should be `TestPatchMatchNetData`

Reply:
agreed, although can we use `TestPatchmatchNetData`? (from the paper)

Reply:
Thanks for updating this Ren